### ISSUE TITLE
Background colors only extend as far as text

### ISSUE THREAD

**Issue by ransomts** [cpaulik/emacs-material-theme#49]
See screenshot for an example, this is happening for headers and blocks, and is reproducible on several machines with this minimal config file. I've added a bunch of spaces to the end of the header here but it happens on headers of all levels. Any ideas how to debug it?
```lisp
(setq custom-theme-directory "~/.emacs.d/packages/emacs-material-theme")
(load-theme 'material-light t)
```
![Screenshot from 2020-11-17 09-38-26](https://user-images.githubusercontent.com/8659901/99404127-3f3daf00-28b9-11eb-9a3d-8c543aa86432.png)

**Comment by cpaulik** [comment]
As far as I know this is just how emacs does it. 

But looking at the first screenshot of e.g. https://github.com/fniessen/emacs-leuven-theme#screenshots 
it seems that the background extends the whole width of the buffer, even if the overline does not.

The second screenshot seems to contradict that though. 

So maybe you can check if `leuven-theme` does it correctly. If it indeed does we should be able to check their config to replicate it for material theme.

**Comment by ransomts** [comment]
Hmm strange it does it for leuven too. For the sake of versions I'm running emacs 27.1 and org 9.3, I wonder if downgrading my org version might help. And yeah looking at those screenshots it doesn't seem very consistent - I suppose I'm more concerned with the src blocks not extending the width than the headers though. Also having a hard time finding another user with this question in general... blarg.

Thanks for the response btw

**Comment by cpaulik** [comment]
I think if you want a definite answer if it is even possible it might be best to inquire on the emacs reddit

**Comment by ransomts** [comment]
so I think I've tracked this down to an issue caused by emacs 27, ([relevant reddit thread](https://www.reddit.com/r/emacs/comments/diahh1/emacs_27_update_changed_how_highlighted_lines/)) making a change to highlighting lines - in particular the new `:extend` attribute.

I can put in a PR with the minimal lines that I got it working for but I'm not sure if that'll maintain the version 24 compatibility you have. In fact I think this would only go back to 26 without raising an error. What do you think?

**Comment by cpaulik** [comment]
I'm happy to include that. So if you could open a PR that would be great.

I'll have to check if backwards compatibility is important or if older emacs versions just get a older version of the theme through MELPA automatically.

**Comment by cpaulik** [comment]
I consider this closed by #50 

Please reopen if other faces need to be fixed
